framework_tvb/tvb/interfaces/web/controllers/users_controller.py

```diff
@@ -146,14 +154,18 @@ def profile(self, logout=False, save=False, **data):
                 common.add2session(common.KEY_USER, self.user_service.get_user_by_id(user.id))
                 common.set_error_message("Could not save changes. Probably wrong old password!!")
         else:
-            #Update session user since disk size might have changed from last time to profile.
+            # Update session user since disk size might have changed from last time to profile.
             user = self.user_service.get_user_by_id(user.id)
             common.add2session(common.KEY_USER, user)
 
         template_specification['user_used_disk_human'] = format_bytes_human(
             self.user_service.compute_user_generated_disk_size(user.id))
         return self.fill_default_attributes(template_specification)
 
+    @cherrypy.expose
+    @using_template('user/silent_check_sso')
+    def check_sso(self):
```

Comment:
What exactly will this check do?
Will we automatically be logged in TVB after integration with HBP Collab, if the current person is already logged in HBP Collab, for example?